### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by rafsalas19**
Document updates for previous PRs

### CODE REVIEW

File: docs/QuickStartGuide.md [hunk 1]
```diff
@@ -14,41 +14,37 @@ Steps
...
-    [all:vars]
-    ansible_user=username
-    ansible_ssh_private_key_file=/path/to/key
-    ansible_ssh_common_args='-o StrictHostKeyChecking=no'
     ```
-    Note: The master node can also be work node as well. The master node will have the Grafana and Prometheus docker containers deployed to it.
+    Note: The manager node can also be a work node as well. The manager node will have the Grafana and Prometheus docker containers deployed to it.
     
-    Note: If you have configured passwordless SSH already, `[all:vars]` section can be skipped.
+    Note: You must have password ssh enabled on your nodes
```
Comment (moes1) [inline docs/QuickStartGuide.md:29]: passwordless ssh?

File: docs/QuickStartGuide.md [hunk 2]
```diff
@@ -14,41 +14,37 @@ Steps
...
     ```sh 
-    python3 moneo.py --deploy -c host.ini full
+    python3 moneo.py --deploy -c hostfile full
     ```
-    note: see usage section of the README doc for more details on Moneo CLI 
-    * manual deployment
-    ```sh
-    ansible-playbook -i host.ini src/ansible/deploy.yaml
-4. Log into the portal by navigating to `http://master-ip-or-domain:3000` and inputting your credentials
+    * If using the Azure HPC/AI market place image or if installation has been performed on all worker nodes by a previous deploymeny we can skip the install step:
```
Comment (moes1) [inline docs/QuickStartGuide.md:38]: market place -> marketplace

File: docs/JobFiltering.md [hunk 3]
```diff
@@ -8,27 +8,18 @@ Steps
 -----
 Moneo must be already deployed and running on the cluster where job grouping will be designated.(Please note that this feature is not supported if Moneo-worker is deployed inside docker, will support it in the future.)
 1. To set a job Id to a subset of nodes a config.ini file with the desired hosts must be passed in:
-*  ```python3 moneo.py -j 4 -c host.ini```
-* The 'host.ini' file will be the same format as the config file used to deploy Moneo for the exception that the worker list will only include the hosts for the job group.
+*  ```python3 moneo.py -j 4 -c hostfile```
+* The 'hostfile' file will be the same format as the config file used to deploy Moneo for the exception that the worker list will only include the hosts for the job group.
```
Comment (moes1) [inline docs/JobFiltering.md:12]: for the exception -> with the exception

**Review by moes1 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -61,16 +61,26 @@ There five categories of metrics that Moneo monitors:
...
 Minimum Requirements
 -----
-
 - python >=3.7 installed
-- docker installed
-- ansible installed (python module)
+
 - OS Support:
-    - Ubuntu 18.04, 20.04
+    - Ubuntu 18.04, 20.04, 22.04
```
Comment (RyoYang) [inline README.md:67]: Do we need to add mariner as well?

File: README.md [hunk 5]
```diff
@@ -61,16 +61,26 @@ There five categories of metrics that Moneo monitors:
...
     - AlmaLinux 8.6
+### Manager node requirements
+- docker
+- parallel-ssh
```
Comment (RyoYang) [inline README.md:71]: do we need to fix a parallel-ssh version as well?

File: README.md (hunk 4 above)
Comment (rafsalas19) [reply README.md:67]: I think that officially we have never tried Moneo on Mariner. So probably not have that here. Even if we use the Mariner docker for geneva agent

File: README.md (hunk 5 above)
Comment (rafsalas19) [reply README.md:71]: yea let me address

Comment (rafsalas19) [reply README.md]: addressed

File: docs/JobFiltering.md (hunk 3 above)
Comment (rafsalas19) [reply docs/JobFiltering.md]: addressed

File: docs/QuickStartGuide.md (hunk 2 above)
Comment (rafsalas19) [reply docs/QuickStartGuide.md]: addressed

File: README.md [hunk 6]
```diff
@@ -83,32 +94,25 @@ git clone https://github.com/Azure/Moneo.git
 cd Moneo
 
 # install dependencies
-python3 -m pip install ansible
+sudo apt-get install pssh
```
Comment (RyoYang) [inline README.md:97]: Do we need to fix pssh version here as well?

Comment (rafsalas19) [reply README.md:97]: good catch

**Pull request merged**